test/e2e/data/e2e_conf.yaml

```diff
@@ -84,14 +84,14 @@ providers:
   - name: aws
     type: InfrastructureProvider
     versions:
-      - name: v1.5.0 # latest published release in the v1beta1 series; this is used for v1beta1 --> v1beta2 clusterctl upgrades test only.
-        value: "https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v1.5.0/infrastructure-components.yaml"
+      - name: v1.5.2 # latest published release in the v1beta1 series; this is used for v1beta1 --> v1beta2 clusterctl upgrades test only.
```

Comment:
This should be `v1.5.1` shouldn't it?

Reply:
We have 1.5.2 release in place, so I thought we shall test with all the changes, shouldn't we? Wdyt?

Reply:
Oh, never mind. I think GitHub has problems. It didn't show the tag or didn't allow me to download this file.

Reply:
Ok cool 🙂